packages/cardano-services/package.json

```diff
@@ -29,7 +29,7 @@
     "cleanup": "rm -rf dist node_modules",
     "cli": "ts-node --transpile-only src/cli.ts",
     "compose:single:up": "yarn compose:up cardano-node ogmios postgres",
-    "compose:up": "__FIX_UMASK__=$(chmod -R a+r ../../compose/placeholder-secrets) docker compose --env-file environments/.env.$NETWORK -p cardano-services-$NETWORK -f docker-compose.yml -f ../../compose/common.yml -f ../../compose/pg-agent.yml -f ../../compose/$(uname -m).yml ${FILES:-}  --profile ${DOCKER_COMPOSE_PROFILE:-none} up",
```

Comment:
Is the strategy to always start blockfrost locally when starting cardano-services?

Reply:
Yes, it is required to test Blockfrost integration also in real networks.